locations/spiders/decathlon.py

```diff
@@ -10,14 +10,21 @@ class Decathlon(CrawlSpider):
     name = "decathlon"
     item_attributes = {"brand": "Decathlon", "brand_wikidata": "Q509349"}
     start_urls = [
+        "https://www.decathlon.be/nl/store-locator",
+        "https://www.decathlon.ch/fr/store-locator",
+        "https://www.decathlon.cz/store-locator",
         "https://www.decathlon.co.uk/store-locator",
         "https://www.decathlon.de/store-locator",
         "https://www.decathlon.es/es/store-locator",
+        "https://www.decathlon.com.hk/en/store-locator",
         "https://www.decathlon.it/store-locator",
         "https://www.decathlon.fr/store-locator",
         "https://www.decathlon.hu/store-locator",
+        "https://www.decathlon.nl/store-locator",
```

Comment:
```suggestion
```

Looks like this one has a different format too, we'll need a different spider for it.